spec/machine/RISCV64/Platform.thy

```diff
@@ -161,6 +161,9 @@ definition irqInvalid :: "irq"
   where
   "irqInvalid \<equiv> 0"
 
+definition timerIRQ :: "irq" where
+  "timerIRQ \<equiv> maxIRQ"
```

Comment:
Is this how it is with RISCV?

Reply:
Yes, that should be correct, at least the C code defines them to [be the same](https://github.com/seL4/seL4/blob/40edd02c220249465ce78515d3493eabd7afc59c/src/arch/riscv/platform_gen.h.in#L36).

Reply:
I see. So, that is how it is in riscv. I guess that should be ok. We are about to add `requalify timerIRQ` in MachineExports.thy, just to mention.

Reply:
That should be fine.